core-processor/src/executor.rs

```diff
@@ -255,7 +345,7 @@ pub fn execute_wasm<A: ProcessorExt + EnvExt + IntoExtInfo + 'static, E: Environ
         }
     };
 
-    log::trace!("term reason = {:?}", termination);
+    log::debug!("term reason = {:?}", termination);
```

Comment:
```suggestion
    log::trace!("term reason = {:?}", termination);
```

Termination reason as clear on upper layers

Reply:
Where?

Reply:
It throws as DispatchResultKind

Reply:
But it's better to show execution term reason in logs